### ISSUE TITLE
Missing parameters list inside parentheses

### ISSUE THREAD

**Issue by FelipeMarcelino** [tenfyzhong/CompleteParameter.vim#23]
# Issue Prelude

- Category
    - [ ] Question
    - [x] Bug
    - [ ] Suggestion
- OS
    - [x] Linux
    - [ ] macOS
    - [ ] Windows
    - [ ] Etc.
- Vim
    - [ ] Vim
    - [x] Neovim
- Language
    - [ ] C
    - [ ] C++
    - [ ] go
    - [x] python
    - [ ] rust
    - [ ] javascript
    - [ ] typescript
    - [ ] erlang

# Completion Engine and Commit SHA
YouCompleteMe:  cd41727  

# Minimal Configuration
inoremap <silent><expr> ( complete_parameter#pre_complete("()")
smap <c-j> <Plug>(complete_parameter#goto_next_parameter)
imap <c-j> <Plug>(complete_parameter#goto_next_parameter)
smap <c-k> <Plug>(complete_parameter#goto_previous_parameter)
imap <c-k> <Plug>(complete_parameter#goto_previous_parameter)

# Issue Details
* What did you do?
>  When i try to autocomplete functions, parameters does not insert inside parentheses.

* What did you expect to happens?
> Insertion of parameters inside function parentheses

* What actually happened?
> Only add empty parentheses

# Messages

[CompleteParameter][03:55:40][TRACE][814724358] {'word': 'XGBClassifier', 'menu': 'class XGBClassifier', 'info': 'XGBClassifier(self, max_depth=3, learning_rate=0.1,
n_estimators=100, silent=True,
objective="binary:logistic",                  nthread=-1, gamma=0,
min_child_weight=1,                  max_delta_step=0, subsample=1,
colsample_bytree=1, colsample_bylevel=1,                  reg_alpha=0,
reg_lambda=1, scale_pos_weight=1,                  base_score=0.5,
seed=0, missing=None)
', 'kind': '', 'abbr': ''}

[CompleteParameter][03:56:19][DEBUG][375168591] []

[CompleteParameter][03:56:43][DEBUG][1816182727] parseds is empty

**Comment by tenfyzhong** [comment]
Can you show me the preview info? Follow next steps to get the preview info. 
1. Edit the python file.
2. Use command mode to enable the preview info: `:set completeopt+=preview`. 
3. Reproduce the completion. And then the preview info will show in the preview window. 

Please copy the info and take a screenshot.
